helpers.js

```diff
@@ -183,12 +203,65 @@ if (argv.secure) secureMode = true;
 /* Is module popular? - for secure mode */
 
 const POPULARITY_THRESHOLD = 10000;
-let isModulePopular = (module) => {
-    let result = request('GET', 'https://api.npmjs.org/downloads/point/last-month/' + module);
-    let downloads = JSON.parse(result.body).downloads;
-    return (downloads > POPULARITY_THRESHOLD);
+let isModulePopular = ({name, dev}) => {
+    let url = 'https://apa.npmjs.org/downloads/point/last-month/' + name;
+    request('GET', url, (error, response, body) => {
+        let downloads = JSON.parse(body).downloads;
+        return (downloads > POPULARITY_THRESHOLD);
+    });
+};
+
+/* Is test file? */
+
+let isTestFile = (name) => {
+    return (name.endsWith('.spec.js') || name.endsWith('.test.js'));
+};
+
+/* Get module names from array of module objects */
+
+let getNamesFromModules = (modules) => {
+    return modules.map(module => module.name);
+}
+
+/* Dedup modules
+ * Divide modules into prod and dev
+ * Deduplicates each list
+ */
```

Comment:
I hate this part of the code, need a more elegant solution
@shubhamgautam Ideas?